templates/registration/logout.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Logout</h1>
+  <form method="post" action="{% url "logout" %}">
```

Comment:
Ensure that the URL name 'logout' matches the URL configuration in your Django project. If you are using Django's built-in authentication views, the URL name should be 'logout' as configured in `urls.py`.